app/Main.hs

```diff
@@ -89,7 +100,7 @@ server =  getUser
 
         postUser :: UserInfo -> Handler PostResponse
         postUser (UserInfo n a) = withSQLite "user-test.sqlite" $ do
-          insert_ users [ User (pack n) a newUserId ]
+          insert_ users [ User (Username (pack n)) a newUserId ]
```

Comment:
I'm wondering if there was a way to avoid having to deal with `SqlType` if we `insert_` a different data type (without using `Username`). 

Side note: should probably also change `UserInfo` to use `Username`.

Reply:
@blinky3713 I've done this specific change in 25d2c73